File: snippets/csharp/VS_Snippets_ADO.NET/Classic WebData ConstraintCollection.IndexOf1 Example/CS/source.cs
using System;
using System.Data;
using System.Windows.Forms;

public class Form1: Form
{
    protected DataSet DataSet1;


    // <Snippet1>
    private void RemoveConstraint(
        ConstraintCollection constraints, Constraint constraint)
    {
        try
        {
            if (constraints.Contains(constraint.ConstraintName) && constraints.CanRemove(constraint)) 
            {
                constraints.RemoveAt(constraints.IndexOf(constraint));
            }
        }
        catch(Exception e) 
        {
            // Process exception and return.
            Console.WriteLine($"Exception of type {e.GetType()} occurred.");
        }
    }
    // </Snippet1>

}

File: snippets/visualbasic/VS_Snippets_ADO.NET/Classic WebData ConstraintCollection.RemoveAt Example/VB/source.vb
Imports System
Imports System.Data
Imports System.Windows.Forms

Public Class Form1
    Inherits Form
    Protected DataSet1 As DataSet
    
' <Snippet1>
Private Sub RemoveConstraint _
    (constraints As ConstraintCollection, constraint As Constraint)

    Try
        If constraints.Contains(constraint.ConstraintName) AndAlso constraints.CanRemove(constraint) Then
            constraints.RemoveAt _
                (constraints.IndexOf(constraint.ConstraintName))
        End If

    Catch e As Exception
        ' Process exception and return.
        Console.WriteLine($"Exception of type {e.GetType().ToString()} occurred.")
    End Try
End Sub
' </Snippet1>
End Class
